### ISSUE TITLE
Minimize button doesn't work in 5.10.1 on Linux Mint

### ISSUE THREAD

**Issue by Jasper557** [mattermost/desktop#3221]
#### Summary
Bug report in one concise sentence.

#### Steps to reproduce
Download version 5.10.1 from this location:
https://releases.mattermost.com/desktop/5.10.1/mattermost-desktop-5.10.1-linux-x64.tar.gz

Extract to any location in your home directory - I will reference it as MT_CLIENT_DIR - the real path doesn't matter

Change permissions to MT_CLIENT_DIR/chrome-sandbox so the Mattermost client does not throw error during startup about incorrect permissions

sudo chown root: ./chrome-sandbox 

sudo chmod 4755 ./chrome-sandbox

Start the application using MT_CLIENT_DIR/mattermost-desktop and log in to your server

Press the "Minimize" button in the top right corner

#### Operating System
Linux Mint

#### Expected behavior
The main Mattermost window gets minimized

#### Observed behavior
The main Mattermost window stays open - it only blinks for a moment

**Comment by devinbinnie** [comment]
@Jasper557 I can't reproduce this one. Which version of Linux Mint are you using/which window environment? Are you on X11 or Wayland?

**Comment by devinbinnie** [comment]
This is likely a window manager issue. Does Cinnamon work?

**Comment by Jasper557** [comment]
Yes, Cinnamon works.

**Comment by devinbinnie** [comment]
@Jasper557 Sounds like a window manager issue then. Did it work in a different version of the Desktop App?

**Comment by Jasper557** [comment]
@devinbinnie Yes, I don't know exactly which version, but in I think 5.9.0 it still worked

**Comment by devinbinnie** [comment]
Can you confirm for me? If there's a regression I'd like to be sure. Thanks :)

**Comment by fallingrock** [comment]
I'm seeing the same thing on Ubuntu 22.04.5 with KDE 5.

**Comment by devinbinnie** [comment]
@Jasper557 Can you try the latest nightly? If it's working there, I might be able to cherry-pick it into the next dot release.

**Comment by Jasper557** [comment]
@devinbinnie it works in the latest nightly :)
